packages/okta-auth-js/test/spec/tokenManager.js

```diff
@@ -932,7 +999,7 @@ describe('TokenManager', function() {
     function localStorageSetup() {
       return setupSync({
         tokenManager: {
-          type: 'localStorage'
```

Comment:
Wait, does this mean these tests haven't been working as intended the whole time?